Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -1483,6 +1495,7 @@ def fetch_incidents(
     """
     # Get the last fetch details, if exist
     limit = min(arg_to_number(demisto.params().get("limit")) or FETCH_MAX_LIMIT, FETCH_MAX_LIMIT)
+    statuses_to_fetch = demisto.params().get("statuses_to_fetch", [])
```

Comment:
Let's add params as an argument to the function and avoid unnecessary calls to `demisto.params()`.